Dockerfile.test.rhel7

```diff
@@ -0,0 +1,85 @@
+# Dockerfile.test.rhel
+#
+# Dockerfile for running tests on RHEL7. To install Python3.9 no RHEL
```

Comment:
"Python3.9 no RHEL"? What does that mean?

Reply:
on